webapi/webapi-webcl-nonw3c-tests/webcl/khronos/conformance/functionalityTesting/buildingAndRunning/setArg_basic.html

```diff
@@ -77,7 +77,7 @@
         var output = wtu.createBuffer(webCLContext, webcl.MEM_WRITE_ONLY, inputArray.BYTES_PER_ELEMENT * resultArray.length);
         wtu.setArg(kernel, 0, input);
         wtu.setArg(kernel, 1, output);
-        wtu.setArg(kernel, 2, new Uint32Array([inputArray.length]));
```

Comment:
@Honry We can can create a Uint32Array in 3 ways:
`new Uint32Array( length );`
`new Uint32Array( array );`
`new Uint32Array( buffer, byteOffset, length );`
So, it should be `new Unit32Array([length])` but not `new Uint32Array([inputArray.length])`. It's the reason that causes the platform crash.
Since the upstream has some updates, I sync the test case to the latest. The other changes are the updates.